scripts/variantstore/wdl/extract/generate_FOFNs_for_import.py

```diff
@@ -6,15 +6,49 @@
 
 import time
 
-# just keep these around as constants for now
-vcf_files_name = "vcf_files.txt"
-vcf_index_files_name = "vcf_index_files.txt"
-sample_names_file_name = "sample_names.txt"
-error_file_name = "errors.txt"
 
 
-# make a default, but allow a user to overwrite it just in case someone wants to tweak this while tuning
-attempts_between_pauses=500
+def get_entities_in_set(data_table_name, sample_set_name):
+    set_of_entities = set()
+    sample_set_list = list(table.list_rows(f"{data_table_name}_set"))
+    for sample_set in sample_set_list:
+        if sample_set.name == sample_set_name: ## else it's some other sample set and we dont care about it
+            for sample in sample_set.attributes['samples']: ## TODO why is this samples in Terra? Is this some Terra hardcoding?!?!
```

Comment:
'samples' looks to be hardcoded in Terra itself rather than entity...do we expect this may change?

Reply:
I don't know and I'm not sure who the right people to ask would be, perhaps a fairly broad internal channel for starters?

Reply:
it's not hardcoded! I ended up just hitting the endpoint